x-pack/plugins/canvas/public/state/selectors/workpad.js

```diff
@@ -90,6 +90,12 @@ export function getElementCounts(state) {
   Object.keys(resolvedArgs).forEach(resolvedArg => {
     const arg = resolvedArgs[resolvedArg];
     const { expressionRenderable } = arg;
+
+    if (!expressionRenderable) {
```

Comment:
nit: Adding a comment here explaining why the value might not exist may be helpful in the future. As a refresher, it's all timing; the `expressionContext` value can exist first, causing there to be a matching key in `resolvedArgs` but no `expressionRenderable` value.